pkg/kubelet/dockershim/helpers_linux_test.go

```diff
@@ -55,26 +58,26 @@ func TestGetSeccompSecurityOpts(t *testing.T) {
 }
 
 func TestLoadSeccompLocalhostProfiles(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "seccomp-local-profile-test")
+	assert.NoError(t, err)
+	defer os.RemoveAll(tmpdir)
+	testProfile := `{"foo": "bar"}`
+	err = ioutil.WriteFile(filepath.Join(tmpdir, "test"), []byte(testProfile), 0644)
+	assert.NoError(t, err)
```

Comment:
shouldn't this be a fatal error?
FYI style guide recommends against assert helpers for exactly this reason, they make it easy to write imprecise tests

Reply:
s/assert/require

Reply:
ack